src/psykal/zero_kern_mod.f90

```diff
@@ -0,0 +1,15 @@
+module zero_kern_mod
+!
+implicit none
+!
+contains
+  !
+  subroutine zero_kern_2d(field,jj,ji)
```

Comment:
AFAICS, this routine isn't used anywhere. Has it been superseded by `zero_layer()`?